### PR TITLE
Add Hafas Logos

### PR DESCRIPTION
**Pull request by kilimnik**
## Proposed change

I added the logos to the intergration Hafas aswell as the virtual integrations.

## Type of change

- [x] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [ ] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/home-assistant/core/pull/78839
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/24193
- Link to integration documentation on our website: https://www.home-assistant.io/integrations/hafas/

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

File: core_brands/db [hunk 1]
```diff
@@ -0,0 +1 @@
+../custom_integrations/deutschebahn
```
Comment (frenck) [inline core_brands/db:1]: This seems incorrect?

**Review by frenck — CHANGES_REQUESTED** [review]
This PR contains multiple changes for different integrations/brands.

Please limits PRs to a single change.

Thanks 👍 

../Frenck

File: core_brands/db (hunk 1 above)
Comment (kilimnik) [reply core_brands/db:1]: Why does it seem incorrect, the pull request in the core adds a virtual integration called db. I thought it wouldn't make any sense to have the db images twice in the repository, therefore I linked them.

**Comment by kilimnik** [comment]
I split the pr into multiple prs.

**Review by frenck — CHANGES_REQUESTED** [review]
This PR adds a brand, however, I do not see multiple uses of that brand. Shouldn't this be a core integration folder instead?

../Frenck

**Comment by kilimnik** [comment]
It wasn't clear to me, what the difference between both folders is, as I did not find any documentation for that. I am happy to move it though

**Comment by frenck** [comment]
> It wasn't clear to me, what the difference between both folders is

The `core_brands` folder is for matching image assets from brand domains in Home Assistant.
The `core_integrations` folder is for matching image assets for integrations domains in Home Assistant.

So, in this case, it depends if you are adding a brand or integration. And if there is a brand added, there should be at least 2 integration available that is categorized under that brand domain (otherwise a branding domain wouldn't be needed).

../Frenck

**Comment by kilimnik** [comment]
Thanks for clarifying, I did change the directory

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**